src/modules/hmis/components/HouseholdStatus.tsx

```diff
@@ -0,0 +1,30 @@
+import { memo } from 'react';
+import EnrollmentStatus from '@/modules/hmis/components/EnrollmentStatus';
+import { HouseholdWithEnrollmentFieldsFragment } from '@/types/gqlTypes';
+
+interface Props {
+  household: HouseholdWithEnrollmentFieldsFragment;
+}
+
+const HouseholdStatus: React.FC<Props> = ({ household }) => {
+  // When showing the status for the household, show the 'most important' status - Incomplete, Open, or Exited.
```

Comment:
This duplicates logic that we already have in the [household view](https://github.com/greenriver/hmis-warehouse/blob/f4a3f8ca1712bd50f53506c178ebb68a2ec59444/db/views/hmis_households_v07.sql#L19). Sorry I did not mention that we already have it! 

I think it would be best to re-use that logic by having the household type resolve an `in_progress` bool (or `any_in_progress`?) that is resolved as `object.any_wip?`. It wouldn't hurt to resolve the other existing fields as well, in case we need them later:
```
household {
   anyInProgress // any_wip
   earliestEntryDate // earliest_entry
   latestExitDate // latest_exit
}
```

Reply:
Good plan, thanks for this suggestion!!